main/guides/getting-started/swaparoo-making-a-payment-explainer.md

```diff
@@ -0,0 +1,41 @@
+# Making Payments in Agoric Smart Contracts
```

Comment:
This isn't really the typical / normal way to make payments in Agoric smart contracts. Normally, contracts don't have direct access to assets. They just tell Zoe how to reallocate assets among seats. This is a special case where a contract isn't in a position to do that, so it has to jump through some special hoops. Also, the payment here is an invitation, so readers looking to send 10 USDC are not going to find an example of how to do that here.

```suggestion
# Sending Invitation Payments using an Address
```

I think the title of the video should change likewise.

Reply:
in preparation for approving, I went back to check the scope of #941. It says:

> Goal: Enhance swaparoo contract to pay out the fee to "someone" e.g. venue operator

We haven't done that. @sufyaankhan do you want to demote that to a stretch goal?
Or should we change this PR from "closes #941" to "refs #941" and make a dapp-agoric-basics ticket to change the contract to pay out the fees?